app/apps/permits/api_queries_decos_join.py

```diff
@@ -354,15 +210,31 @@ def get_documents_with_folder_id(self, folder_id):
     def _get_decos_folder(self, decos_object):
         if not settings.USE_DECOS_MOCK_DATA:
             try:
-                decos_object_id = decos_object["content"][0]["key"]
+                #  Get all Decos object id's
+                list_of_decos_object_ids = [
+                    obj["key"] for obj in decos_object["content"]
+                ]
             except (KeyError, IndexError):
-                decos_object_id = False
+                list_of_decos_object_ids = False
```

Comment:
Hierboven is list_of_decos_object_ids een list, maar hier een boolean. Kan het hier niet een lege list worden zodat het data type hetzelfde is